frontend/src/components/Discover_Event/EventDetail.tsx

```diff
@@ -17,70 +18,111 @@ const EventDetail: React.FC = () => {
   };
 
   const onCheckOut = (eventData: EventType) => {
-    console.log(eventData);
+    console.log("Checking out", eventData);
   };
 
   return (
-    <div className="flex flex-col md:flex-row p-6">
-      <div className="flex-1">
+    <div className="event-detail-container p-6">
+      <div className="event-image pb-7">
         <img
           src={eventData.imageUrls}
           alt={eventData.title}
-          className="rounded-lg mb-4"
+          className=" event-image rounded-lg  mb-4"
         />
-        <h1 className="text-2xl font-bold mb-2">{eventData.title}</h1>
-        <p className="text-gray-600 mb-2">{eventData.location}</p>
-        <p className="text-gray-600 mb-2">{eventData.startTime}</p>
-        <p className="text-gray-600 mb-2">{eventData.endTime}</p>
-        <div className="mb-4">
-          {eventTags.map((tag) => (
-            <span
-              key={eventData.id}
-              className="inline-block bg-blue-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2 mb-2"
-            >
-              {tag}
-            </span>
-          ))}
-        </div>
-        <h2 className="text-xl font-semibold mb-2">About this event</h2>
-        <p>{eventData.description}</p>
       </div>
-      <div className="flex-1">
-        <div className="bg-gray-100 p-6 rounded-lg shadow-lg">
-          <h3 className="text-lg font-semibold mb-2">Tickets</h3>
-          <div className="flex items-center mb-4">
-            <button
-              onClick={() => handleTicketAmountChange(-1)}
-              className="text-gray-500 focus:outline-none focus:bg-gray-300 p-2 rounded-l-md"
-              disabled={ticketAmount <= 1}
-              aria-label="Decrease ticket amount"
-            >
-              -
-            </button>
-            <span className="flex items-center justify-center bg-white text-black mx-2 p-2">
-              {ticketAmount}
-            </span>
-            <button
-              onClick={() => handleTicketAmountChange(1)}
-              className="text-gray-500 focus:outline-none focus:bg-gray-300 p-2 rounded-r-md"
-              aria-label="Increase ticket amount"
-            >
-              +
-            </button>
+      <div className="md:flex-row grid grid-cols-4 gap-2 event-info-container">
+        <div className=" event-info col-span-3">
+          <h1 className="text-2xl font-bold mb-2">{eventData.title}</h1>
+          <p className="text-gray-600 mb-2">{eventData.location}</p>
+          <p className="text-gray-600 mb-2">{eventData.startTime}</p>
```

Comment:
I noticed we have three of the same details about the event, is this intentional or should be remove to only has one also we could made the  picture into a fix sized that look good and not required audience to scroll down in other to see information. Since if the picture height is large then it does not look good.
![Screenshot 2024-05-23 at 7 29 11 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/60468960-7416-4313-966e-b1c7ca53b12f)
![Screenshot 2024-05-23 at 7 30 47 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/0be7b4af-68ad-4ead-892e-d987bfa3840f)
ssets/69278249/62129fd6-927b-4b9d-89ce-772e2fe226b3)
![Screenshot 2024-05-23 at 7 31 37 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/58d84ca4-10a2-460c-aef3-e2e591f538d4)